packages/consent/consent-tools/src/domain/validation/options-validators.ts

```diff
@@ -58,11 +58,7 @@ export function validateAnalyticsInstance(
   analytics: unknown
 ): asserts analytics is AnyAnalytics {
   assertIsObject(analytics, 'analytics')
-  if (
-    'load' in analytics &&
-    'on' in analytics &&
-    'addSourceMiddleware' in analytics
-  ) {
+  if ('load' in analytics && 'addSourceMiddleware' in analytics) {
```

Comment:
Do we need to check for `addDestinationMiddleware` now too?

Reply:
Err, yeah -- this typeguard is incomplete =) Just here as kind of a sanity check.